flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/example/IncrementalSSSP.java

```diff
@@ -110,24 +105,20 @@ public static void main(String [] args) throws Exception {
 			// Emit results
 			if(fileOutput) {
 				resultedVertices.writeAsCsv(outputPath, "\n", ",");
-
-				// since file sinks are lazy, we trigger the execution explicitly
-				env.execute("Incremental SSSP Example");
 			} else {
 				resultedVertices.print();
 			}
 
+			env.execute("Incremental SSSP Example");
```

Comment:
I'm not sure whether I am missing something... Why do you add `env.execute()` after `print()`. 
It's no longer needed. Have a look here:
https://github.com/apache/flink/blob/master/flink-examples/flink-java-examples/src/main/java/org/apache/flink/examples/java/graph/PageRankBasic.java